src/bindings/python/nixl-meta/nixl_meta_utils.py

```
@@ -0,0 +1,97 @@
+    if cuda_bindings is not None:
+        version = major(getattr(cuda_bindings, "__version__", None))
+        if version is not None:
+            return version
+
+    # cupy already imported: use it.
+    cupy = sys.modules.get("cupy")
+    if cupy is not None:
+        try:
+            return cupy.cuda.runtime.runtimeGetVersion() // 1000
+        except Exception:
+            pass
```

Ruff (0.15.20)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)